scripts/ci/run-integration-tests.sh

```diff
@@ -21,7 +21,7 @@ main () {
 
     ./mypy.sh
     ./pylint.sh
-    ./run_tests.sh --log-cli-level=ERROR --mount-dir="$TEMP_RESOURCES_DIR"
+    ./run_tests.sh --log-cli-level=ERROR --mount-dir="$TEMP_RESOURCES_DIR" -n 4 --dist=loadscope
```

Comment:
From https://pypi.org/project/pytest-xdist/ :

> --dist=loadscope: tests will be grouped by module for test functions and by class for test methods, then each group will be sent to an available worker, guaranteeing that all tests in a group run in the same process. This can be useful if you have expensive module-level or class-level fixtures. Currently the groupings can’t be customized, with grouping by class takes priority over grouping by module. This feature was added in version 1.19.

Wasn't aware of this setting  👍